### PR TITLE
docs: add preview images and chore: add Nord theme

### PR DESCRIPTION
**Pull request by hydino2085143**
68e5102c0491420b1b488958491fd175610860bb adds preview images to all the themes, for a good browsing/theme selection experience.
 
3eccd1fcb40f1356c4b9030383392b8447b34d0b adds my Nord theme to this repository under the `GPL-3.0-or-later` license. 

It might be best to separate this PR into two, so that it is more organized.

### CODE REVIEW

File: themes/Nord/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+SPDX-FileCopyrightText: 2022 hydino2085143 <priya.aravamuthan@protonmail.com>
+
+SPDX-License-Identifier: CC0-1.
```
Comment (DioEgizio) [inline themes/Nord/README.md:4]:
```suggestion
SPDX-License-Identifier: CC0-1.0

```

**Review by DioEgizio — COMMENTED** [review]
please add licenses to images too to comply with reuse

**Comment by hydino2085143** [comment]
What image license should I use? Is CC-1.0 fine?

**Comment by DioEgizio** [comment]
CC0-1.0 is fine yeah

**Review by DioEgizio — COMMENTED** [review]
Add newline and you forgot to rename the Dracula png license file

**Comment by hydino2085143** [comment]
> Add newline and you forgot to rename the Dracula png license file

Where should I add newline?

**Comment by hydino2085143** [comment]
My fork is quite messed up right now, give me some time

File: themes/Breeze-Dark/preview-breeze-dark.png.license [hunk 2]
```diff
@@ -0,0 +1,7 @@
+SPDX-FileCopyrightText: 2022 hydino2085143
+
+<<<<<<< HEAD
+SPDX-License-Identifier: CC0-1.0
+=======
+SPDX-License-Identifier: CC0-1.0
+>>>>>>> 1e662db4e7542198637403313bfdb641b79065db
```
Comment (DioEgizio) [inline themes/Breeze-Dark/preview-breeze-dark.png.license:7]: I think you did an oopsie here

File: themes/Catppuccin-Frappe/preview-catppuccin-frappe.png.license [hunk 3]
```diff
@@ -0,0 +1,7 @@
+SPDX-FileCopyrightText: 2022 hydino2085143
+
+<<<<<<< HEAD
+SPDX-License-Identifier: CC0-1.0
+=======
+SPDX-License-Identifier: CC0-1.0
+>>>>>>> 1e662db4e7542198637403313bfdb641b79065db
```
Comment (DioEgizio) [inline themes/Catppuccin-Frappe/preview-catppuccin-frappe.png.license:7]: .

File: themes/Catppuccin-Latte/preview-catppuccin-latte.png.license [hunk 4]
```diff
@@ -0,0 +1,7 @@
+SPDX-FileCopyrightText: 2022 hydino2085143
+
+<<<<<<< HEAD
+SPDX-License-Identifier: CC0-1.0
+=======
+SPDX-License-Identifier: CC0-1.0
+>>>>>>> 1e662db4e7542198637403313bfdb641b79065db
```
Comment (DioEgizio) [inline themes/Catppuccin-Latte/preview-catppuccin-latte.png.license:7]: .

File: themes/Catppuccin-Macchiato/preview-catppuccin-macchiato.png.license [hunk 5]
```diff
@@ -0,0 +1,7 @@
+SPDX-FileCopyrightText: 2022 hydino2085143
+
+<<<<<<< HEAD
+SPDX-License-Identifier: CC0-1.0
+=======
+SPDX-License-Identifier: CC0-1.0
+>>>>>>> 1e662db4e7542198637403313bfdb641b79065db
```
Comment (DioEgizio) [inline themes/Catppuccin-Macchiato/preview-catppuccin-macchiato.png.license:7]: .

File: themes/Nord/themeStyle.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+ * SPDX-FileCopyrightText: 2022 hydino2085143
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+QToolTip { 
+	color: #D8DEE9; 
+	background-color: #3B4252;
+	border: 1px solid #3B4252
+}
```
Comment (DioEgizio) [inline themes/Nord/themeStyle.css:11]: Add the newline

**Review by DioEgizio — APPROVED** [review]
Looks good to me

**Pull request merged**